src/molecules/product-list-item/ProductListItem.html

```diff
@@ -0,0 +1,128 @@
+<a-list-item tag="div" class="a-product-list-item">
+  <div class="a-product-list-item__image">
+    <!--@slot Custom product image (Scoped slot)-->
+    <slot
+      name="image"
+      :alt="name"
+      :image-url="imageUrl"
+    >
+      <a-image
+        :src="imageUrl"
+        :alt="name"
+      />
+    </slot>
+  </div>
+  <div class="a-product-list-item__info">
+    <div class="a-product-list-item__data">
+      <!--@slot Custom product name (Scoped slot)-->
+      <slot
+        name="name"
+        :product-name="name"
+        :product-url="url"
+      >
+        <router-link
+          :to="url"
+          class="link link--inverted a-product-list-item__name"
+        >
+          {{ name }}
+        </router-link>
+      </slot>
+      <!--@slot Custom product price (Scoped slot)-->
+      <slot
+        name="price"
+        :price="price"
+        :special-price="specialPrice"
+        :old-price="oldPrice"
+      >
+        <a-price
+          :price="price"
+          :special-price="specialPrice"
+          :old-price="oldPrice"
+          class="a-product-list-item__price-wrapper"
+          old-price-class="a-product-list-item__price-wrapper--old"
+          special-price-class="a-product-list-item__price-wrapper--special"
+        />
+      </slot>
+
+      <a-list
+        v-if="options"
+        tag="dl"
+        class="a-product-list-item__attributes"
+      >
+        <template v-for="item in options">
+          <!-- @slot Custom product item option (Scoped slot) -->
+          <slot
+            name="option"
+            :option="item"
+          >
+            <a-list-item tag="dt">
+              {{ item.label }}
+            </a-list-item>
+            <a-list-item tag="dd">
+              {{ item.value }}
+            </a-list-item>
+          </slot>
+        </template>
+      </a-list>
+    </div>
+
+    <div
+      :class="[
+        'a-product-list-item__actions',
+        { 'a-product-list-item__actions--to-right': hideQty }
+      ]"
+    >
+      <div
+        v-if="!hideQty"
+        class="a-product-list-item__qty-wrapper"
+      >
+        <!--@slot Custom product quantity (Scoped slot)-->
+        <slot
+          name="quantity"
+          :item-qty="qty"
+          :change-qty="onChange"
+        >
+          <a-quantity-update
```

Comment:
What do you think about using quantity as not default element? the same as here https://github.com/SnowdogApps/alpaca-ui/blob/develop/src/03-modules/product-item/ProductItem.html

Add just quantity slot and use if needed

Reply:
and also please check styles b/c qty label is on the left. I think we can add default variant story, cart variant, with slots at least

Reply:
Maybe it's better to move these variants directly to qty component? I mean add modifiers to specify different label positions - `--top`, `--left` etc

Reply:
I'll check this